package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cioos-metadata-entry-form",
-  "homepage": "https://cioos-siooc.github.io/metadata-entry-form/",
+  "homepage": ".",
```

Comment:
I'm not sure about this line. it makes the paths relative to the index.html file in the build folder. This makes the preview deployments work but I'm not sure if it will be a problem for github pages deployment? @n-a-t-e do you know?